x/pool-incentives/keeper/keeper_test.go

```diff
@@ -52,6 +52,7 @@ func (suite *KeeperTestSuite) preparePoolWithPoolParams(poolParams gammtypes.Poo
 			suite.ctx,
 			acc,
 			sdk.NewCoins(
+				sdk.NewCoin("uosmo", sdk.NewInt(10000000000)),
```

Comment:
This is just a drive-by change right? Didn't really affect anything relative to this PR AFAICT. (If so, thats fine)